ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleAdapter.java

```diff
@@ -1,10 +1,169 @@
-package com.vungle.mediation;
-
-import com.google.ads.mediation.vungle.VungleMediationAdapter;
-import com.google.android.gms.ads.mediation.MediationAdapter;
-
-/**
- * A {@link MediationAdapter} to load and show Vungle rewarded video ads using
- * Google Mobile Ads SDK mediation.
- */
-public class VungleAdapter extends VungleMediationAdapter { }
+package com.vungle.mediation;
+
+import android.support.annotation.Keep;
+
+import com.google.ads.mediation.vungle.VungleMediationAdapter;
+import com.google.android.gms.ads.mediation.MediationAdapter;
+
+/**
+ * A {@link MediationAdapter} to load and show Vungle rewarded video ads using
+ * Google Mobile Ads SDK mediation.
+ */
+@Keep
+public class VungleAdapter extends VungleMediationAdapter { }
+
+//TODO cleanup left it for cross-reference during merge

```

Comment:
this was accidentally not cleaned-up after some big merge we did and initially we kept it for cross-referencing, we should keep this file as it was earlier, and remove all the commented out code part
Am not sure if we can completely remove it though ! , I think we reference this class in AdMob dashboard, and thats why we need to still keep it, even if its derived without any overrides.

Reply:
Done. Removed all commented out code.